news/2020-02-18-mappers-report-february.md

```diff
@@ -0,0 +1,178 @@
+
+### osu!mania
+
+- [**Hitsound Additions Requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.4): Following this change, osu!mania beatmapsets containing only difficulties of Insane or above do not require hitsound additions. 
+
+- [**Difficulty name requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!mania#rules):
+A new specification has been added that sets with a single key mode must not use the key mode in the difficulty name.
+
+- [**Long note changes in Easy and Normal**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!mania#-easy):
+The previous guidelines in Easy and Normal that "A long note cannot be released during another long note's body" have been removed. They have been replaced with "long note releases should be at least 1/1 beat from each other" for Easy difficulties and "long note releases should be at least 1/2 beat from each other" for Normal difficulties.
+
+
+
+## <a id="natmeetings"></a>NAT Meetings
+
+The past 2 months have been filled with public meetings organized by the NAT, aimed at improving the ranking system based on community input. A wide range of topics such as BN/NAT activity requirements and improving the transparency of the application process to the two groups has been covered so far, and there is more to come! If you wish to participate in future meetings, join the [osu!dev discord server](https://discord.gg/ppy) and go through the verification process.
+
+Upcoming meetings will be announced with a specific time and agenda in the #modding channel of the osu!dev Discord server, as well as in an announcement post in the [Mapping Discussion](https://osu.ppy.sh/community/forums/56) forum. The next meeting is scheduled for 1st of March, but the topic and agenda are still being worked on and will be announced soon.	
+
+### [Jan 3rd, 2020 - Promotion of NAT Members](https://osu.ppy.sh/community/forums/topics/1006119)
+
+The most important information which was discussed during this meeting was the addition of new NAT and its team size. The NAT wiki article has also been updated to include relevant information such as how the team size of the NAT depends on what is needed and in what work fields. If more help in one work field is required, a new NAT will be considered. 
+
+### [Jan 18th, 2020 - Application Evaluation](https://osu.ppy.sh/community/forums/topics/1012519)
+
+
+The BN application mod requirements have been made more clear, and the BN website also received several improvements such. One addition was a “Would you nominate this map, if no why?” checkbox. The modder can explain in this section whether they would nominate the beatmaps which they provided as modding examples. All this information helps the NAT and evaluating BNs to make a fair evaluation of the applicant. 
```

Comment:
```suggestion
The BN application mod requirements have been made more clear, and the BN website also received several improvements, the biggest one being a “Would you nominate this map. If not, why?” checkbox on the application form. It vastly improves the fair evaluation system.
```
Shortened this one as the purpose of checkbox thing is pretty much self-explanatory so there's probably no point in giving it such a complex description.